src/gobprepare/config.py

```diff
@@ -8,6 +8,11 @@
 POSTGRES_DRIVER = 'postgresql'
 DEBUG = os.getenv('GOB_DEBUG', False)
 
+STREAMING_GRAPHQL_ENDPOINT = '/gob/graphql/streaming/'
+GRAPHQL_ENDPOINT = '/gob/graphql/'
+WRITE_BATCH_SIZE = 10000
+DEFAULT_API_HOST = 'http://localhost:8141'
+
```

Comment:
Why not set GOB_API_HOST here so that all config (including read from env) is located at one place.

Reply:
will do.

Reply:
done